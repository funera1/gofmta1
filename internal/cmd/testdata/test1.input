package main

/*
[google]https://google.com
	func main() {
        var a int;;
	}
*/
func main() {
	var a int
}
