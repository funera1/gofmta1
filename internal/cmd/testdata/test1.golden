package main

/*
cmnt

	func test1() {
		var a int
	}
*/
func main() {
	var a int
}
